src/App.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
 import { Routes, Route } from 'react-router-dom';
 import { useDispatch } from 'react-redux';
 import { useEffect } from 'react';
```

Comment:
- [ ]  Kindly as stated by the previous reviewer 😃 Good job, cards are displaying correctly on a mobile screen. However, the styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that the cards on your list follow the alternating dual-color pattern. Each cell of the list item should have surrounding cells having different (or slightly alternating color shade) background color(s).💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
| ![image](https://github.com/citec-47/React-capstone/assets/83360636/8dad6ccf-79c3-44e8-87a0-6ba1ad06bd30)   |   ![image](https://github.com/citec-47/React-capstone/assets/83360636/463472be-482c-4877-8f72-22d62c3bda70)|